spark-cassandra-connector-embedded/src/main/scala/com/datastax/spark/connector/embedded/EmbeddedCassandra.scala

```diff
@@ -23,10 +39,25 @@ trait EmbeddedCassandra {
     require(hosts.isEmpty || configTemplates.size <= hosts.size,
       "Configuration templates can't be more than the number of specified hosts")
 
+
+    val templateDir = {
```

Comment:
The yamls should only be breaking on Major's from here on out (3+), but previously (2.2=-) they broke on minor prior to that.